### PR TITLE
With apollo example

### PR DESCRIPTION
**Pull request by brunorzn**

### CODE REVIEW

File: examples/with-apollo/src/Home.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import React, { Component } from 'react';
+import logo from './react.svg';
+import './Home.css';
+import { Link } from 'react-router-dom';
+
+class Home extends Component {
+  static async getInitialProps({ req, res, match, history, location, ...ctx }) {
+    return { stuff: 'whatevs' };
```
Comment (jaredpalmer) [inline examples/with-apollo/src/Home.js]: Can we fetch some data from a public graphql server and render the results?

Comment (brunorzn) [reply examples/with-apollo/src/Home.js]: Yes. I'm on it

**Comment by willopez** [comment]
@jaredpalmer other than the conflicts, is there anything else needed to move this PR forward? I am willing to help out if necessary, as I am interested in using after with apollo.

**Comment by brunorzn** [comment]
@willopez I've resolved the conflicts ; you can still use the example directly from the fork :

https://github.com/brunorzn/after.js/tree/with-apollo-example/examples/with-apollo

Tell me if you have any trouble setting it up.

**Comment by willopez** [comment]
@brunorzn thanks man! I'll give it a go.

**Comment by willopez** [comment]
Hi @brunorzn I followed the instructions in the `README` inside the `with-apollo` folder and got this error: 
```
λ yarn start
yarn run v1.5.1
$ razzle start
 WAIT  Compiling...

 ERROR  Failed to compile CLIENT with 2 errors

./src/client.js
Module not found: Can't resolve '@jaredpalmer/after' in '/Users/will/code/after.js/examples/with-apollo/src'

 ERROR  Failed to compile CLIENT with 2 errors

./src/routes.js
Module not found: Can't resolve '@jaredpalmer/after' in '/Users/will/code/after.js/examples/with-apollo/src'```

Any ideas why?

**Comment by brunorzn** [comment]
@willopez try `yarn add @jaredpalmer/after`

**Comment by willopez** [comment]
@brunorzn that fixed it, thank you.

**Comment by tzarger** [comment]
@jaredpalmer Any plans to merge this?

File: examples/with-apollo/src/Document.jsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    // get attributes from React Helmet
+    const htmlAttrs = helmet.htmlAttributes.toComponent();
+    const bodyAttrs = helmet.bodyAttributes.toComponent();
+    return (
+      <html {...htmlAttrs}>
+        <head>
+          <meta httpEquiv="X-UA-Compatible" content="IE=edge" />
+          <meta charSet="utf-8" />
+          <title>After with Apollo !</title>
+          <meta name="viewport" content="width=device-width, initial-scale=1" />
```
Comment (HofmannZ) [inline examples/with-apollo/src/Document.jsx]: Meta tags are duplicate

File: examples/with-apollo/src/About.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+      <div>
+        {film.title} has be directed by {film.director}
+      </div>
+      <Link to="/">Home -></Link>
+    </div>
+  );
+}
+
+export default graphql(QUERY, {
+  options: ({ match: { params: { id } } }) => ({ variables: { id } }),
```
Comment (HofmannZ) [inline examples/with-apollo/src/About.js]: Maybe use the new `react-apollo` syntax (aka the Query and Mutation components)

**Comment by jaredpalmer** [comment]
Does this work with the latest release?

**Comment by suciuvlad** [comment]
@jaredpalmer I tried it with v1.3.1 and it works. Looking forward to the merge 👍

**Comment by suciuvlad** [comment]
@jaredpalmer I've tried this with v1.3.1 and it's working. Looking forward to the merge 👍

**Comment by smkhalsa** [comment]
@jaredpalmer anything else needed for this to get merged?

**Pull request closed without merging**